### ISSUE TITLE
在codellama上微调的性能没有提升

### ISSUE THREAD

**Issue by HPRCEST** [codefuse-ai/MFTCoder#18]
在codellama-7b-instruct上进行微调，使用的数据就是当前开源的数据，跑了4个epoch，其余的参数设置都沿用的默认的，但是在humaneval和mbpp上的表现都下降了，请问这是什么原因呢？

**Comment by twelveand0** [comment]
首先，我们一般不选择在instruct版模型上进行二次微调，一般是选择base模型为底座进行微调。其次，影响微调效果有很多因素，例如，learning rate、global batch size、lora settings、train/eval比例等

**Comment by lwh8915** [comment]
@HPRCEST ,你好，可以求一份微调codellama的代码吗，我是一个初学者，想要参考学习一下
